samples/ocp/1.6.1/storage_csm_powerflex_v2110.yaml

```diff
@@ -0,0 +1,406 @@
+apiVersion: storage.dell.com/v1
```

Comment:
I suggest to remove this file as this should  be updated only after we publish the images to redhat ..only then  we get the SHA ids for the changed images

Reply:
@rajendraindukuri : The file was added because we had to update the new SHA of SDC 4.5.2.1, this file has the updated SHA

Reply:
Removed the ocp/.1.6.1 samples as the YAMLs will be updated after the image is published to Red Hat by @rajendraindukuri